wire-ios-system/Scripts/config.txt

```diff
@@ -1,5 +0,0 @@
-framework=WireSystem
-repository=wire-ios-system
-organization=wireapp
-platforms=osx,ios
-release-name-format={PROPER_NAME} the {ADJECTIVE} {MONSTER} from {GERMAN_TOWN}
```

Comment:
We have files like this (Scripts/config.txt) in several projects.
Can those be deleted?

Reply:
Assuming yes since https://github.com/wireapp/wire-ios/pull/2423 was approved.